src/main/kotlin/com/faforever/userservice/hydra/HydraService.kt

```diff
@@ -155,4 +155,18 @@ class HydraService(
             .bodyValue(error)
             .retrieve()
             .bodyToMono(RedirectResponse::class.java)
+
+    fun revokeAllSessions(revokeRefreshTokensRequest: RevokeRefreshTokensRequest): Mono<RedirectResponse> {
```

Comment:
rename Sessions in method name too please

Reply:
DOne